### DISCUSSION TITLE
Feedback for course "Practical work 1"

### DISCUSSION THREAD

**Discussion by ludelafo** [heig-vd-dai-course/heig-vd-dai-course#5] [Polls]
Can you let us know what was easy and what was difficult for you during this chapter?

This will help us to improve the course and adapt the content to your needs. If we notice some difficulties, we will come back to you to help you.

You can use reactions to express your opinion on a comment! 🚀

Vous pouvez aussi nous laisser votre avis en français, il n'y a pas de souci !

**Comment by AlexandreIorio** [comment]
Hi, 
I have a quick question about the following points: 

4	The entire team contributes to the project and can explain it	0.2
22	Everyone talks during the presentation	0.2

What happens if we work alone?

Thank you !

**Reply by ludelafo** [reply]
Hello !

Thank you for your comment.

We have updated the criteria as follows:

> 4\. The whole team contributes to the project and can explain it in detail

This implicitly means that if you're working alone, we expect you to explain in detail how you created your application, more so than if you're working in a group where explanations don't need to be too detailed to give each member equal speaking time.
 
> 22\. Everyone speaks during the presentation, and the presentation lasts the time allowed

This implicitly means that you have to keep to the time allowed, neither too little nor too much, and that it can be a challenge to keep to these times, even when working alone or in a group.

Is it more clear now?

**Reply by AlexandreIorio** [reply]
Yes, thank you !

**Reply by ludelafo** [reply]
Perfect!

I will tag all students so everyone should receive the notification regarding this point. I will also talk about this next time in class.

@heig-vd-dai-course/students
@heig-vd-dai-course/teaching-staff

**Comment by jonas-tr** [comment]
Hello,

I have a question about issues. 

Do we create an issue only for bugs and problems or can we create issues for missing features for example?

Thanks!

**Reply by ludelafo** [reply]
Hello!

We think it is good practice to create issues for (missing) features as well so it encourages you to split the work in a rational matter ("step 1: Create the README, step 2: Add the IntelliJ + Maven project, step 3: Add picocli dependency, etc."). It will help you organize your work, even more if you work in group.

In my opinion, the term "issue" is misleading; I try to create issues for anything related to my projects (ideas, new features, bugs, human workflows, technical workflows, etc.) so I can organize myself and don't forget anything related to the project I'm working on.

Does it answer your question?

**Reply by jonas-tr** [reply]
Yes, thanks for the quick answer!

**Reply by ludelafo** [reply]
You're welcome!

**Comment by ludelafo** [comment]
All previous comments have been marked as hidden (outdated) for the start of the new DAI semester.
